src/js/modules/add.js

```diff
@@ -0,0 +1,18 @@
+const add = () => {
```

Comment:
- [ ] OPTIONAL
Your input element is accepting empty fields. Kindly ensure to validate this input to check that the input is not empty before data is stored.

**Your project looks like this, when empty details is entered:**
![image](https://user-images.githubusercontent.com/12745474/130165071-b193f763-7827-4ccd-a59e-f084e42968c4.png)

Reply:
thank you its a great advice, i will change it :smile: